roles/build_dependencies/README.md

```diff
@@ -0,0 +1,35 @@
+Role Name
+=========
+
+This role builds dependencies for CDH applications. 
+
+Requirements
+------------
+
+
+Role Variables
+--------------
+
+
+Dependencies
+------------
+
+
+Example Playbook
+----------------
+
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+
+    - hosts: servers
+      roles:
+         - { role: username.rolename, x: 42 }
```

Comment:
is this boiler plate that hasn't been filled in yet? even if no reqs/deps shouldn't we say that explicitly?

Reply:
This is boiler plate.